specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -0,0 +1,340 @@
+                         "policy"
+                        ],
+                "summary": "Sets the policy for a given kind of TEE.",
+                "operationId": "Policy_Set",
+                "consumes": [ "text/plain" ],
+                "parameters": [
+                        {
+                            "$ref": "#/parameters/ApiVersionParameter"
+                        },
+                        {
+                            "$ref": "#/parameters/AuthorizationParameter"
+                        },
+                        {
+                            "$ref": "#/parameters/TeeKind"
+                        },
+                        {
+                            "name": "newAttestationPolicy",
+                            "in": "body",
+                            "description": "JWT Expressing the new policy",
+                            "required": true,
+                            "x-ms-client-flatten": true,
+                            "schema": {
+                                "type": "string"
+                            }
+                        }
+                    ],
+                "responses": {
+                    "200": {
+                        "description": "Success"
+                    },
+                    "400": {
+                        "description": "Bad request",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    },
+                    "401": {
+                        "description": "Request is unauthorized",
+                        "schema": {
+                            "type": "string"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Set Policy": {         
+                      "$ref": "./examples/Set_Policy.json"         
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                         "attestation",
+                         "policy"
+                        ],
+                "summary": "Deletes the attestation policy for the specified tenant and reverts to the default policy.",
+                "operationId": "Policy_Delete",
+                "consumes": [ "text/plain" ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AuthorizationParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/TeeKind"
+                    },
+                    {
```

Comment:
Body parameters in DELETE requests have no defined semantics. The expectation is that the resource indicated in the request URL would be deleted. Can you please expand on why you need a body parameter here?

Reply:
Because we need to have evidence that the client has permissions to delete the resource (AAD bearer tokens are not secure so cannot be used). The expression of "evidence that the client has permissions to delete the resource" is contained in the body of the other REST APIs and this makes them consistent.

Reply:
Could you make this a header?